pages/1.14/administering-clusters/draining-a-node/index.md

```diff
@@ -0,0 +1,77 @@
+
+<p class="message--warning"><strong>WARNING: </strong>All tasks that are running on the agent will be killed when draining is initiated. Use this feature with caution!</p>
+
+Initiating draining will cause all tasks on the target agent node to receive a kill event immediately, assuming the agent is currently reachable. If the agent is unreachable, initiation of the kill event will be delayed until the agent is reachable by the master again. When the tasks receive a kill event, a SIGTERM signal will be sent to the task to begin the killing process. Depending on the particular task's behavior, this signal may be sufficient to terminate it. Some tasks may use this signal to begin the process of graceful termination, which may take some time. After some delay, a SIGKILL signal will be sent to the task, which forcefully terminates the task if it is still running. The delay between the SIGTERM and SIGKILL signals is determined by the length of the task's kill grace period. If no grace period is set for the task, a default value of several seconds will be used.
+
+### Prerequisites:
+
+*   [DC/OS CLI installed and configured](/1.14/cli/).
+*   Sufficient [permissions](/1.14/security/ent/perms-reference) to drain nodes (Enterprise DC/OS only).
+
+### Maximum Grace Period
+
+It is possible to set a max grace period when draining, which sets an upper bound on the duration of the kill grace period. When this is set, the minimum of the task's grace period and the max grace period will be used when killing the task.
+
+### Decommissioning an Agent After Draining
+
+In some cases, such as autoscaled cloud instances which are later scaled down, you may know that after the node is drained and taken down, it will never come back again. Each agent node is uniquely identified by metadata on its hard disk, so even if other instances are scaled up in the future, if they don't retain the persisted metadata of the previous instances, they are viewed as new nodes by the DC/OS cluster.
+
+In cases where the unique agent will not return after it is drained and removed from the cluster, you can specify an option which will automatically decommission the node after it has finished draining, which removes the agent's metadata from the cluster. If this option is not specified, the cluster will remember the agent node in case it reregisters with the cluster later on.
+
+## Draining with the CLI
+
+To drain a node with the DC/OS CLI, use the `dcos node drain` command:
+
+    ```bash
+    dcos node drain <mesos-id>
+    ```
+
+The maximum grace period and/or decommission options may be specified like so:
+
+    ```bash
+    dcos node drain <mesos-id> --timeout=10m --decommission
+    ```
+
+Once draining on an agent has begun, you can monitor the tasks on that node by running:
+
+    ```bash
+    dcos task list --agent-id <mesos-id>
```

Comment:
Should we rewrite `<mesos-id>` as `<mesos-agent-id>` throughout this entire doc?